### ISSUE TITLE
TK3.1 - Rail zone micro journey

### ISSUE THREAD

**Issue by KaterinaKir** [wmcadigital/wmn-sprints#67]
**Epic:** Help users to understand which rail zones they need

***
#### As a
commuter, leisure, business 

#### I need to
understand which rail zones I need

#### So that
I can buy a ticket that suits my needs

***
#### What is a rail zone?
- [x] User can understand an explanation of what rail zones are, written in plain English

#### Which rail zones do I need?
- [x] User can find out which zones two or more train stations are in
- [x] User can see which stations are outside of these zones (you need zones 1-4, do you ever go to these stations in zone 5?)

##### Map view
- [x] User can see a schematic map of rail zones
- [x] User can see if a train station has parking
- [x] User can see if a train station has disabled access
- [x] User can filter stations that have parking
- [x] User can filter stations that have disabled access

##### List view
- [x] User can see a list of train stations and their rail zones
- [x] User can see if a train station has parking
- [x] User can see if a train station has disabled access
- [x] User can filter stations that have parking
- [x] User can filter stations that have disabled access

#### Technical
- [x] When a user has chosen zones, they are remembered on the ticket page

***

**Comment by mrmjprice** [comment]
Most recent map asset: https://www.westmidlandsrailway.co.uk/sites/default/files/assets/download_ct/20200219/QQdp1ULWJTTEJ2QAdzAHlZ6ezQEJ9j262l5mZZDorJs/wmr_regional_map_a3_feb20.pdf

**Comment by mrmjprice** [comment]
This might help: https://www.npmjs.com/package/react-svg-map

**Comment by mrmjprice** [comment]
Will need integrating to #76

**Comment by mrmjprice** [comment]
`{
  "railStationAccess": [
    {
      "crsCode": "ACG",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "ADD",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "ALB",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "ALV",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "AST",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "ATH",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "BTG",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "BER",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "BEH",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "BKW",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "BEP",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "BSC",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "BBK",
      "parking": false,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "BMO",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "BNS",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "BSW",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "BKT",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "BKD",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "BLX",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "BWN",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "BBS",
      "parking": false,
      "stepFreeAccess": false
    },
    {
      "crsCode": "BRV",
      "parking": false,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "BMV",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "BUL",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "CNL",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "CAO",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "CRD",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "CLV",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "CSL",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "CEH",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "CWL",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "CSY",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "COS",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "COV",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "CAA",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "CRA",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "DZY",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "DDG",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "DTW",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "DUD",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "DDP",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "EWD",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "ERD",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "FWY",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "FOK",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "GVH",
      "parking": false,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "GMV",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "HAG",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "HLG",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "HIA",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "HSD",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "HBY",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "HTN",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "HNF",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "HNL",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "HFD",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "HNK",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "JEQ",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "KNW",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "KID",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "KNN",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "LAW",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "LGG",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "LPW",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "LEH",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "LMS",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "LED",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "LIC",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "LTV",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "LBK",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "LOB",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "LYE",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "MVL",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "MGN",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "NBR",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "NMP",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "NFD",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "NUN",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "OKN",
      "parking": false,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "OHL",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "OLT",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "PKG",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "PRY",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "PSW",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "RDC",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "ROW",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "RUG",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "RGT",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "RGL",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "SAD",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SLY",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "SEN",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "SFN",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "SRL",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SHR",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SMA",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "SGB",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SMR",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "SOL",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SRI",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "STA",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SCF",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SBJ",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SBT",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SAV",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "STY",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "SUT",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "TAB",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "TAM",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "TFC",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "THW",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "TLK",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "THL",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "TIP",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "TYS",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "UNI",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "WSL",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "WRW",
      "parking": true,
      "stepFreeAccess": null
    },
    {
      "crsCode": "WRP",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "WTO",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "WLN",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "WTE",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "WMR",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "WMC",
      "parking": false,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "WNE",
      "parking": false,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "WTT",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "WVH",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "WDE",
      "parking": false,
      "stepFreeAccess": null
    },
    {
      "crsCode": "WWW",
      "parking": false,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "WOF",
      "parking": false,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "WOS",
      "parking": true,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "WOP",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "WYL",
      "parking": true,
      "stepFreeAccess": "full"
    },
    {
      "crsCode": "WYT",
      "parking": false,
      "stepFreeAccess": "partial"
    },
    {
      "crsCode": "YRD",
      "parking": true,
      "stepFreeAccess": "full"
    }
  ]
}
`
